### ISSUE TITLE
Can't get the face of the block clicked on PlayerBucketEvent

### ISSUE THREAD

**Issue by thebigsmileXD** [PocketMine/PocketMine-MP#3918]
It should return the block face.
But doesn't. So you can't give the activated face to a plugin calling PlayerInteractEvent.
See here:
```php
    public function onBukkitEvent(PlayerBucketEvent $event){
        /*
         * Disables bucket in plots the player doesn't own
         */
    	$ev = new PlayerInteractEvent($event->getPlayer(), $event->getBucket(), $event->getBlockClicked(), $event->getFace());
    	$this->onEventOnBlock($ev);
    }
```
Maybe add getBlockFace() to the PlayerBucketEvent. Thanks

**Comment by PEMapModder** [comment]
Did you mean PlayerBucketEvent's subclasses? PlayerBucketEvent itself doesn't contain a handlerList.

**Comment by thebigsmileXD** [comment]
@PEMapModder first of all.. why can you close?
2nd: Why don't you wait for an answer.
3rd: I mean the PlayerBucketFillEvent and the PlayerBucketEmptyEvent, that was correct. Thats why i said Event***s***.

**Comment by PEMapModder** [comment]
@thebigsmileXD already fixed in d538a25, anyway.

**Comment by thebigsmileXD** [comment]
I asked more, not if it is fixed!
